packages/client/test/sim/README.md

```diff
@@ -0,0 +1,45 @@
+### Shandong sim setup
+
+This sim test is to setup a single ethereumjs<>lodestar instance to allow executing testvectors for the EIPs targetting Shanghai hardfork.
+
+### Prerequisite(s)
+
+- Docker since lodestar docker image is used to run CL to drive the post-merge sim run
+- Docker should run without `sudo`, else prefix `sudo` in from of docker commands in `test/sim/single-run.sh`
+
+### How to run
+
+1. `npm install` the `ethereumjs-monorepo`
+2. `cd packages/client`
+3. `docker pull chainsafe/lodestar:latest`
+4. Install package `jq` is not installed
+5. Create a data directory `data/shandong` (or any other place convinient for you).
+
+Currently you can just start the local instance by
+`DATADIR=/data/shandong test/sim/./single-run.sh`
```

Comment:
@acolytec3 this is where you would want to update without the `/`, right?

Reply:
updated